src/Metadata/Resource/Factory/AnnotationResourceMetadataFactory.php

```diff
@@ -54,7 +54,11 @@ public function create(string $resourceClass): ResourceMetadata
             return $this->handleNotFound($parentResourceMetadata, $resourceClass);
         }
 
-        $resourceAnnotation = $this->reader->getClassAnnotation($reflectionClass, ApiResource::class);
+        $resourceAnnotation = $this->reader->getClassAnnotation(
+            $reflectionClass,
+            ApiResource::class
+        );
```

Comment:
Readability I'd say. To keep [lines 80 characters or less](http://www.php-fig.org/psr/psr-2/#overview). It does help me a lot :-)

But ok, going back to one line.